applications/GeoMechanicsApplication/custom_constitutive/linear_elastic_plane_strain_2D_law.cpp

```diff
@@ -89,12 +101,84 @@ void GeoLinearElasticPlaneStrain2DLaw::CalculatePK2Stress(const Vector& rStrainV
 {
     KRATOS_TRY
 
+    UpdateInternalDeltaStrainVector(rValues);
+
     Matrix C;
     this->CalculateElasticMatrix(C, rValues);
-    // Total formulation
-    noalias(rStressVector) = prod(C, rStrainVector);
+
+    // Incremental formulation
+    noalias(mStressVector) = mStressVectorFinalized + prod(C, mDeltaStrainVector);
```

Comment:
Do you think we can add a unit test for this functionality (of course that also serves as documentation, that makes it easier to understand)

Reply:
That would indeed be nice, I would appreciate help with it.

Reply:
Did this work out in the end?